Update guide.txt (#4)
Vehmloewff/jikno-backend-api: guide.txt

     ________________                 ________
     |______   _____|                /  ____  \
            |  |                    /  /    \  \
            |  |                   /  /      \  \ 
            |  |                  /  /        \  \        ______    _________
            |  |                 /  /__________\  \       |     |       |
    ____    |  |                /  _____________   \      |_____/       |
    |  |    |  |               /  /              \  \     |             |
    |  |____/  /   ___        /  /                \  \    |             |
    |_________/   |___|      /  /                  \  \   |        _____|_____

A Guide to the Jikno.com user API:

	Basic Syntax:

> 	jikno.com/api/api.php?key=<your_api_key>&action=<action>

Reviewer: Sorry I didn't tell you this before, but the url is incomplete:
```suggestion
	https://www.jikno.com/api/api.php?key=<your_api_key>&action=<action>
```

				}
			}

> 		The manager by curriculum.jikno.com has it's own branch, the to do list it's own, and so on.

Reviewer: This sounds a bit funny...
You might want to consider one of the following:
```suggestion
		The manager at manager.jikno.com has it's own branch, the to do list it's own, and so on.
```
or
```suggestion
		The manager by Jikno has it's own branch, the to do list it's own, and so on.
```

> 		The manager by curriculum.jikno.com has it's own branch, the to do list it's own, and so on.

Reviewer: This sounds a bit funny...
You might want to consider one of the following:
```suggestion
		The manager at manager.jikno.com has it's own branch, the to do list it's own, and so on.
```
or
```suggestion
		The manager by Jikno has it's own branch, the to do list it's own, and so on.
```
